apps/daimo-mobile/src/view/sheet/CreateBackupSheet.tsx

```diff
@@ -0,0 +1,204 @@
+import { SlotType, assert, findAccountUnusedSlot } from "@daimo/common";
+import Octicons from "@expo/vector-icons/Octicons";
+import { useCallback, useContext, useMemo, useState } from "react";
+import { Platform, StyleSheet, View } from "react-native";
+import Animated, { FadeIn, FadeOut } from "react-native-reanimated";
+import { useSafeAreaFrame } from "react-native-safe-area-context";
+
+import { DispatcherContext } from "../../action/dispatch";
+import { useNav } from "../../common/nav";
+import { useAccount } from "../../logic/accountManager";
+import { AddKeySlotButton } from "../screen/keyRotation/AddKeySlotButton";
+import { Badge } from "../shared/Badge";
+import { ButtonBig } from "../shared/Button";
+import { OctName } from "../shared/InputBig";
+import { ScreenHeader } from "../shared/ScreenHeader";
+import Spacer from "../shared/Spacer";
+import { color } from "../shared/style";
+import { TextBody, TextCenter, TextH3 } from "../shared/text";
+
+export function CreateBackupSheet() {
+  const [step, setStep] = useState<0 | 1>(0);
+
+  return (
+    <View style={{ paddingBottom: 36 }}>
+      {step === 0 ? (
+        <CreateBackupContent setStep={setStep} />
+      ) : (
+        <OfflineBackupContent setStep={setStep} />
+      )}
+    </View>
+  );
+}
+
+function AddPasskeyButton() {
+  const account = useAccount();
+  assert(account != null);
+
+  const passkeySlot = useMemo(
+    () => findAccountUnusedSlot(account, SlotType.PasskeyBackup),
+    []
+  );
+
+  return (
+    <AddKeySlotButton
+      buttonTitle="Backup with Passkey"
+      account={account}
+      slot={passkeySlot}
+    />
+  );
+}
+
+function AddSecurityKeyButton() {
```

Comment:
these two buttons can be merged easily? param SlotType

Reply:
done